Cdn_Plugin_WidgetMaxCdn.php

```diff
@@ -39,8 +39,8 @@ function run() {
 		add_action( 'w3tc_ajax_cdn_maxcdn_widgetdata', array(
 			$this, 'w3tc_ajax_cdn_maxcdn_widgetdata' ) );
 
-		if ( $this->have_zone && $this->authorized && isset( $_GET['page'] ) &&
-				strpos( $_GET['page'], 'w3tc_dashboard' ) !== false ) {
+		if ( $this->have_zone && $this->authorized && isset( Util_Request::get_string( 'page' ) ) &&
```

Comment:
Changed to ! empty and created a variable to avoid 2 function calls. However this conditional check has no action block defined so it's an empty if statement? I assume this was added with the intention of expanding functionality

Reply:
Removed variable and condition block.